js/main.js

```diff
@@ -21,4 +22,84 @@ function save(event) {
   data.entries.unshift(formInput);
   $photoImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   $formJournal.reset();
+  $entryList.appendChild(renderEntry(data.entries[0]));
+  viewSwap('entries');
+  if ($noEntries.classList.length === 0) {
+    toggleNoEntries();
+  }
 }
+
+// Viewing Entries
+
+function renderEntry(entry) {
+  var $listEntry = document.createElement('li');
+  $listEntry.setAttribute('class', 'row');
+  var $photoDiv = document.createElement('div');
+  var $image = document.createElement('img');
+  $photoDiv.setAttribute('class', 'column-half entry-photos');
+  $image.setAttribute('src', entry.photo);
+  $photoDiv.appendChild($image);
+  $listEntry.appendChild($photoDiv);
+  var $textDiv = document.createElement('div');
+  $textDiv.setAttribute('class', 'column-half');
+  var $entryTitle = document.createElement('h3');
+  $entryTitle.textContent = entry.title;
+  $textDiv.appendChild($entryTitle);
+  var $paragraph = document.createElement('p');
+  $paragraph.textContent = entry.notes;
+  $textDiv.appendChild($paragraph);
+  $listEntry.appendChild($textDiv);
+  return $listEntry;
+}
+
+var $entryList = document.querySelector('ul');
+
+document.addEventListener('DOMContentLoaded', function () {
+  for (var i = 0; i < data.entries.length; i++) {
+    $entryList.appendChild(renderEntry(data.entries[i]));
```

Comment:
You should be using `prepend()` here, so that the newest entry will always show up first!